generic/tests/netperf.py

```diff
@@ -106,6 +105,37 @@ def env_setup(session, ip, user, port, password):
         remote.scp_to_remote(ip, shell_port, username, password,
                              agent_path, "/tmp")
 
+    def mtu_set(mtu):
+        """
+        Set server/client/host's mtu
+
+        :param mtu: mtu value to be set
+        """
+
+        linux_mtu_cmd = params.get("linux_mtu_cmd")
+        windows_mtu_cmd = params.get("windows_mtu_cmd")
+        error_context.context("Changing the MTU of guest", logging.info)
+        if params.get("os_type") == "linux":
+            ethname = utils_net.get_linux_ifname(server_ctl, mac)
+            ssh_cmd(server_ctl, linux_mtu_cmd % (ethname, mtu))
+        elif params.get("os_type") == "windows":
+            connection_id = utils_net.get_windows_nic_attribute(
+                            server_ctl, "macaddress", mac, "netconnectionid")
+            ssh_cmd(server_ctl, windows_mtu_cmd % (connection_id, mtu))
+
+        error_context.context("Changing the MTU of client", logging.info)
+        ssh_cmd(client, linux_mtu_cmd % (params.get("client_physical_nic"), mtu))
+
+        netdst = params.get("netdst", "switch")
+        host_bridges = utils_net.Bridge()
+        br_in_use = host_bridges.list_br()
+        if netdst in br_in_use:
+            ifaces_in_use = host_bridges.list_iface()
+            target_ifaces = list(ifaces_in_use + br_in_use)
+        error_context.context("Change all Bridge NICs MTU to %s" % mtu, logging.info)
```

Comment:
Is this change MTU of host? It's more readable to add "host" in context. Because you write "change MTU of guest /client" above.